app/v1/connectors/RetrieveUkSavingsAccountAnnualSummaryConnector.scala

```diff
@@ -38,12 +38,15 @@ class RetrieveUkSavingsAccountAnnualSummaryConnector @Inject() (val http: HttpCl
 
     val nino           = request.nino.nino
     val incomeSourceId = request.savingsAccountId
+    val path = s"income-tax/nino/$nino/income-source/savings/annual/${request.taxYear.asDownstream}?incomeSourceId=$incomeSourceId"
```

Comment:
Maybe move path closer to where it is used (see comment on CreateAmendUk..Connector).